public/index.html

```diff
@@ -15,7 +15,7 @@
       img-src * data:;
       style-src-elem 'self' 'unsafe-inline' https://fonts.googleapis.com https://kit-free.fontawesome.com https://unpkg.com;
       script-src-elem 'self' 'unsafe-inline' https://cdn.lr-ingest.io https://kit.fontawesome.com https://js.stripe.com https://unpkg.com;
-      style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://kit-free.fontawesome.com;
+      style-src 'self' 'unsafe-inline' https://fonts.googleapis.com https://kit-free.fontawesome.com https://unpkg.com;
```

Comment:
Ah, taking another look at these, I think both the style and script src content CSP lines need to be more specific for graphiql